src/services/dashboard.js

```diff
@@ -24,17 +24,14 @@ function scrollUntilEmpty(data) {
 
 export async function queryControllers(params) {
   try {
-    const { selectedDateRange } = params;
+    const { selectedDateRange, username } = params;
 
     return request.post(`${endpoints.pbench_server}/controllers/list`, {
       data: {
-        user: 'username', // TODO: Will need to get user context here
+        user: username || 'public',
```

Comment:
This is hardcoding a user name that might change.  We should let the server side API code handle and defaults.

Reply:
Yes, that's a good point; we can default a missing `username` in the payload to `public` and simply omit it here or give a null value.

Reply:
+1